BGDIINF_SB-1901: Added geoadmin README and deploy script

ltshb opened the pull request.

ltshb committed:
BGDIINF_SB-1901: Added geoadmin README and deploy script

boecklic commented on geoadmin/README.md:
```diff
@@ -0,0 +1,31 @@
+this branch should be kept up to date with the fork master branch.
+
+Any geoadmin changes should be done in `geoadmin_master` branch. This branch follows the general geoadmin GIT FLOW rules.
+To get the lastest changes from the radiantearth, we first merge radiantearth master branch into our master branch and
```
We might not be able to adopt any upstream change right away. We'll probably stay on the current version until version 3 is release... maybe make a comment that new upstream versions should be thoroughly tested before merged into geoadmin_master

ltshb replied:
done

boecklic commented on geoadmin/deploy.sh:
```diff
@@ -0,0 +1,101 @@
+    s3 sync \
+        --delete \
+        --cache-control "public, max-age=${MAX_AGE}" \
+        dist/  s3://${S3_BUCKET_NAME}/
```
does this take into account the `browser/` prefix? (see ticket)

ltshb replied:
done

ltshb committed:
Fixed deployment on /browser

Now the files are deployed with the `browser` prefix.

Use the landing page (catalog) and not the collections endpoint.

Also removed the STAC_VERSION which is not required as the version is
taken from the catalog response.

Added as well the HISTORY_MODE=hash which is specified in the README.md
when deploying on S3 or Apache.

ltshb committed:
Updated geoadmin readme.

boecklic reviewed (approved):
:1st_place_medal: except that it doesn't work yet on prod: data.geo.admin.ch/browser/index.html is showing an error
